sys/net/application_layer/emcute/emcute.c

```diff
@@ -128,7 +128,7 @@ static int syncsend(uint8_t resp, size_t len, bool unlock)
             DEBUG("[emcute] syncsend: got response [%i]\n", result);
             xtimer_remove(&timer);
             res = result;
-            retries = EMCUTE_N_RETRY;
+            retries = (EMCUTE_N_RETRY + 1);
```

Comment:
this breaks out of the for loop? why not ```break```?

Reply:
also possible. Will fix